src/tests/class_tests/openms/source/IsotopeDistribution_test.cpp

```diff
@@ -161,6 +161,39 @@ START_SECTION(IsotopeDistribution& operator *= (Size factor))
 		TEST_REAL_SIMILAR(id.getContainer()[i].second, container[i].second)
 	}
 
+  // test gapped isotope distributions, e.g. bromide 79,81 (missing 80)
+  {
+    EmpiricalFormula ef("Br2");
+    IsotopeDistribution id = ef.getIsotopeDistribution(5);
+    container.clear();
+    container.push_back(make_pair<Size, double>(158, 0.2569476));
+    container.push_back(make_pair<Size, double>(159, 0.0));
+    container.push_back(make_pair<Size, double>(160, 0.49990478));
+    container.push_back(make_pair<Size, double>(161, 0.0));
+    container.push_back(make_pair<Size, double>(162, 0.24314761));
```

Comment:
could you please add some data where you got these numbers, how you calculated the "true" numbers here so that future generations can easily figure out what you did here?